helpers/lbryApi.js

```diff
@@ -1,15 +1,27 @@
 var axios = require('axios');
+var db = require("../models");
 
 module.exports = {
-	publishClaim: function(publishParams){
+	publishClaim: function(publishParams, fileType){
 		var deferred = new Promise(function(resolve, reject){
 			console.log(">> lbryApi >> publishClaim:", publishParams);
 			axios.post('http://localhost:5279/lbryapi', {
 				"method": "publish", 
 				"params": publishParams
 			}).then(function (response) {
-				console.log(">> 'publish' success");
-				resolve(response.data);
+				console.log(">> 'publish' success", response);
+				db.File.create({
+					name: publishParams.name,
+					claim_id: response.data.result.claim_id,
+					outpoint: response.data.result.outpoint,
+					file_name: "test",
+					file_path: publishParams.filePath,
+					file_type: fileType,
+					nsfw: nsfw,
+				}).catch(function(error){
+					console.log('An error occurred when writing to the MySQL database:', error);
```

Comment:
It might be a good idea to declare an extremely simple error logging class or function that simply calls `console.log` internally. That way when you want to replace it later with more sophisticated logging, you can do so easily, rather than needing to track down all the log calls.

Not a blocking suggestion.